docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -626,30 +617,106 @@ These steps can also be performed through BitBucket's UI wizard. This UI wizard
+5. Optional: Create a separate CI job for diff-aware scanning, which scans only changed files in PRs or MRs, by repeating steps 1-3 and uncommenting the `SEMGREP_BASELINE_REF` definition provided within the code snippet.
+
+<Tabs
+    defaultValue="buildkite-semgrep"
+    values={[
+    {label: 'CI with Semgrep App', value: 'buildkite-semgrep'},
+    {label: 'Stand-alone CI job', value: 'buildkite-standalone'},
+    ]}
+>
+
+<TabItem value='buildkite-semgrep'>
+
+```yaml
 - label: ":semgrep: Semgrep"
   commands:
-    - export SEMGREP_REPO_URL="$(echo "$BUILDKITE_REPO" | sed -e 's#.\{4\}$##')"
-    - export SEMGREP_BRANCH=${BUILDKITE_BRANCH}
-    - export SEMGREP_COMMIT=${BUILDKITE_COMMIT}
-    - export SEMGREP_PR_ID=${BUILDKITE_PULL_REQUEST}
-    - echo "$BUILDKITE_REPO" | sed 's#https://github.com/##' | sed 's#.git##'
-    - export SEMGREP_REPO_NAME="$(echo "$BUILDKITE_REPO" | sed -e 's#https://github.com/##' | sed -e 's#.git##')"
+
+    # Uncomment the following line to scan changed 
+    # files in PRs or MRs (diff-aware scanning): 
+    # - export SEMGREP_BASELINE_REF = "main"
+    
+    # Troubleshooting:
+
+    # Uncomment the following lines if Semgrep App > Findings Page does not create links
+    # to the code that generated a finding or if you are not receiving PR or MR comments.
+    # - export SEMGREP_COMMIT=${BUILDKITE_COMMIT}
+    # - export SEMGREP_PR_ID=${BUILDKITE_PULL_REQUEST}
+    # - export SEMGREP_BRANCH=${BUILDKITE_BRANCH}
+    # - export SEMGREP_REPO_URL="$(echo "$BUILDKITE_REPO" | sed -e 's#.\{4\}$##')"
+    # - echo "$BUILDKITE_REPO" | sed 's#https://github.com/##' | sed 's#.git##'
+    # - export SEMGREP_REPO_NAME="$(echo "$BUILDKITE_REPO" | sed -e 's#https://github.com/##' | sed -e 's#.git##')"
+    
     - semgrep ci 
   
   plugins:
     - docker#v3.7.0:
         image: returntocorp/semgrep
         environment:
-          # Scan with rules set in Semgrep App's rule board
-          # Make a token at semgrep.dev/orgs/-/settings/tokens
+          # The following variable is required for a Semgrep App-connected scan:
           - "SEMGREP_APP_TOKEN"
 ```
 
+</TabItem>
 
-## CircleCI
+<TabItem value='buildkite-standalone'>
 
+```yaml
+- label: ":semgrep: Semgrep"
+  commands:
+    # Define rules to scan with by setting the SEMGREP_RULES environment variable. 
+    - export SEMGREP_RULES="p/default"
+
+    # To scan changed files in PRs or MRs (diff-aware scanning):
+    # - export SEMGREP_BASELINE_REF=${BUILDKITE_BRANCH}
+
+    - semgrep ci 
+  
+  plugins:
+    - docker#v3.7.0:
+      image: returntocorp/semgrep
 ```
+</TabItem>
+
+</Tabs>
+
+## CircleCI
+
+To add Semgrep into your CircleCI pipeline:
+
+1. Create or edit your `config.yml` configuration file in the repository you want to scan.
+2. Copy the relevant code snippet provided after these instructions.
```

Comment:
Same as with BuildKite. Which code snippet? If it is too big, we can make it a subheader and link to it. (Like `### CircleCI configuration example` etc) Why not include the code snippet here under step 2? Or is it the last step we have to do after we follow all instructions?

Reply:
(Done making header for all CI providers.)